analyses/independent-samples/run-independent-samples.sh

```diff
@@ -16,12 +16,12 @@ cd "$(dirname "${BASH_SOURCE[0]}")"
 
 Rscript -e "rmarkdown::render('00-repeated-samples.Rmd',params=list(base_run = ${RUN_FOR_SUBTYPING}), clean = TRUE)"
 
-if [[ RUN_FOR_SUBTYPING == "0" ]]
-then
+#if [[ RUN_FOR_SUBTYPING == "0" ]]
+#then
    HISTOLOGY_FILE="../../data/histologies.tsv" 
-else 
-   HISTOLOGY_FILE="../../data/histologies-base.tsv"  
-fi
+#else 
+#   HISTOLOGY_FILE="../../data/histologies-base.tsv"  
+#fi
```

Comment:
Can we remove the comments from the code here as well?

Reply:
Done

Reply:
`998 WGS primary specimens
181 WGS secondary specimens
1126 WGS specimens (including non-primary)
1014 WGS+WXS+Panel primary specimens
181 WGS+WXS+Panel secondary specimens
1142 WGS+WXS+Panel specimens (including non-primary)` This is the output - I don't think the number changed from the original - the original results were from wc -l so the header was counted